README.md

```diff
@@ -4,6 +4,21 @@
 
 **QA: Shreyas Sabnis**
 
```

Comment:
Include code to install dependencies using the requirements.txt

Reply:
This has been add. Please review. Thanks!